src/app/pages/owner/owner.page.ts

```diff
@@ -33,7 +35,18 @@ export class OwnerPage implements OnInit {
       }
       this.waitingFlag = false;
     })
-  }
 
+    this.waitingFlag = true;
+
+    this.commonSer.getCollectionList('worker').then(cloudSalesData => {
+      this.workers = cloudSalesData; 
```

Comment:
rename cloudSalesData to cloudWorkerProductionData. it will work but good to give correct name